feathr_project/feathr/utils/job_utils.py

```diff
@@ -82,6 +90,9 @@ def get_result_df(
     Returns:
         Either Spark or pandas DataFrame.
     """
+    if format is not None:
+        data_format = format
```

Comment:
If we decide to deprecate `format` argument in the future, a warning like `this argument will be deprecated, please upgrade to data_format....` maybe useful.